Fakebook.Profile/Fakebook.Profile.UnitTests/APITests/ProfileIntegrationTests.cs

```diff
@@ -1,7 +1,115 @@
+
+namespace Fakebook.Profile.UnitTests.APITests
 {
     public class ProfileIntegrationTests
     {
+        [Fact]
+        public async Task ProfileRepoAddProfile()
+        {
+            using var contextFactory = new ContextFactory();
+            using var context = contextFactory.CreateContext();
+
+            DomainProfile test = new DomainProfile("tdunbar@google.com", "Trevor", "Dunbar");
+            test.BirthDate = new DateTime(1994, 6, 30, 0, 0, 0, 0, DateTimeKind.Unspecified);
+
+            var repo = new ProfileRepository(context);
+            await repo.CreateProfileAsync(test);
+
+            var compare = context.EntityProfiles.Local.Single(x => x.Id == 3); //ID is 3 because there's already 2 users in seed data
+            Assert.Equal(test.Email, compare.Email);
+            Assert.Equal(test.FirstName, compare.FirstName);
+            Assert.Equal(test.LastName, compare.LastName);
+        }
+
+        [Fact]
+        public async Task ProfileRepoGetProfile()
+        {
+            using var contextFactory = new ContextFactory();
+            using var context = contextFactory.CreateContext();
+
+            var repo = new ProfileRepository(context);
+            var test = await repo.GetProfileAsync("john.werner@revature.net");
+
+            var compare = context.EntityProfiles.Local.Single(x => x.Id == 1); //ID is 3 because there's already 2 users in seed data
+            Assert.Equal(test.Email, compare.Email);
+            Assert.Equal(test.FirstName, compare.FirstName);
+            Assert.Equal(test.LastName, compare.LastName);
+        }
+
+        [Fact]
+        public async Task ProfileRepoTestFollowers()
+        {
+            using var contextFactory = new ContextFactory();
+            using var context = contextFactory.CreateContext();
+
+            DomainProfile test = new DomainProfile("tdunbar@google.com", "Trevor", "Dunbar");
+            test.BirthDate = new DateTime(1994, 6, 30, 0, 0, 0, 0, DateTimeKind.Unspecified);
+
+            DomainProfile testFriend = new DomainProfile("friend@google.com", "Friendly", "Person");
+            test.BirthDate = new DateTime(1994, 6, 30, 0, 0, 0, 0, DateTimeKind.Unspecified);
+
+            var repo = new ProfileRepository(context);
+            await repo.CreateProfileAsync(test); // ID 3
+            await repo.CreateProfileAsync(testFriend); // ID 4
+
+            test.FollowingEmails.Add("friend@google.com");
+            testFriend.FollowerEmails.Add("tdunbar@google.com");
+
+            testFriend.FollowingEmails.Add("tdunbar@google.com");
+            test.FollowerEmails.Add("friend@google.com"); // 2 accounts that follow each other
+
+            await repo.UpdateProfileAsync(test.Email, test);
+            await repo.UpdateProfileAsync(testFriend.Email, testFriend);
+
+            var compareTest = context.EntityProfiles.Local.Single(x => x.Id == 3);
+            var compareTestFriend = context.EntityProfiles.Local.Single(x => x.Id == 4);
```

Comment:
if the repository is supposed to be doing savechanges, then we should test that it is.
if it's not, then we should stop it.
if this PR is anticipating a future change of that, then i guess that's ok.

Reply:
@tdunbar123 what's the answer here

Reply:
Gotta look through trainer code because I forget how we tested savechanges

Reply:
Still a little confused about how I should test this if the update method is calling savechanges...

Reply:
Maybe this test is redundant because the one below it is essentially doing the same thing?